desc/geometry/curve.py

```diff
@@ -220,7 +220,7 @@ def from_input_file(cls, path):
             inputs = InputReader.parse_vmec_inputs(f)[-1]
         else:
             inputs = InputReader().parse_inputs(f)[-1]
-        curve = cls(
+        curve = FourierRZCurve(
```

Comment:
is there a reason to avoid using cls?

Reply:
The coil class inherits this method but I wasn't sure if we wanted to treat the axis as a coil

Reply:
Ah ok I see